### PR TITLE
docs: add drafts for moderation guide

### PR DESCRIPTION
**Pull request by nhcarrigan**
#### What does this PR do?

This pull request adds drafts for some moderation/maintainer guidelines to facilitate our community volunteers.

#### Description of the task to be completed?

We'll likely want to iterate over this as a team a couple of times, to ensure that our maintainers have a reference for frequently asked questions.

#### How can this be manually tested?

Read it, I guess?

#### Any background context you want to provide?

I was naughty and discussed this with Eddie in DMs. 😬 

#### Is there any relevant issue to this PR?

Nope.

#### Screenshots (if appropriate)

No screenshots, but here's a reminder to have fun today!

danganronpa-celestia

#### Questions

How much wood would a woodchuck chuck if a woodchuck could chuck wood?

### CODE REVIEW

File: moderators/github.md
Comment (krishguptadev) [inline moderators/github.md:9]: I would also suggest adding branch protection rules for the same 🤔

File: moderators/github.md
Comment (krishguptadev) [inline moderators/github.md:46]: How about configure stale bot it self to do this 🤔

File: moderators/github.md
Comment (nhcarrigan) [reply moderators/github.md:9]: That's something Eddie has to set up. We don't have access to the settings.

Additionally, there might be times where we want to allow single-review merges. 🤷🏻

File: moderators/github.md
Comment (eddiejaoude) [inline moderators/github.md:49]: Yes 👍 it can scare people off and I know none of us want this

**Review by eddiejaoude — DISMISSED** [review]
This is great ❤️

File: moderators/github.md
Comment (nhcarrigan) [reply moderators/github.md:46]: I'm personally against stale bot in general, but *especially* for pull requests.

Folks may have become distracted with another project, and the last thing we should do is have a bot send an automated message and dismiss their efforts.

**Review by krishguptadev — DISMISSED** [review]

File: moderators/github.md
Comment (Panquesito7) [inline moderators/github.md:55]: In some repositories I maintain, we have a `dont-close` label which tells the bot *not* to close that issue/PR, which works well to keep important issues/PRs open without having to remove the label many times. 🙂

**Review by Panquesito7 — COMMENTED** [review]
Great work on this, Nick! 🚀 I could help with the moderation guidelines as well 😄 just ping me in Discord and let's get working. 👍

File: moderators/github.md
Comment (nhcarrigan) [reply moderators/github.md:55]: If we're doing that, the purpose of the bot is null.

Honestly I'm still in favour of removing it entirely and doing something like community issue triage sessions. But that's all out of scope for this PR 🙂

Comment (Panquesito7) [reply moderators/github.md:55]: "Null"? Well, that will prevent unnecessary noise from maintainers 😅 I've also seen many important issues being closed and never opened again.

Comment (nhcarrigan) [reply moderators/github.md]: [I've opened a discussion to continue this conversation](https://github.com/EddieHubCommunity/support/discussions/2977).

**Review by krishguptadev — DISMISSED** [review]
Q: Are these changes fixed for markdownlint from #2976

**Comment by nhcarrigan** [comment]
The action is passing, so yes 🙂

File: docs/moderators/github.md
Comment (schmelto) [inline docs/moderators/github.md:47]:
```suggestion
  - Be sure to leave a friendly message thanking them for their efforts and inviting them to request that we re-open the PR when they are available to complete the changes.
```

**Review by schmelto — DISMISSED** [review]
For the first draft it's looking quite nice to me. Therfore the approve from my side is there. 

Should we add a paragraph explaining some specialty for PRs by a repository maintainer?

Maybe also what to do with quick fixes like running a Linter on the project thus the ci is not failing anymore?

**Review by krishguptadev — APPROVED** [review]

File: docs/moderators/github.md
Comment (kaiwalyakoparkar) [inline docs/moderators/github.md:9]: I guess @nhcarrigan mentioned about 1 approval to merge above. So should we have 1 here?

Comment (krishguptadev) [reply docs/moderators/github.md:9]: None other than maintainers can merge, so a merge is considered a approval 🤔

Comment (Vyvy-vi) [reply docs/moderators/github.md:9]: I think we can also have a provision of -> Many approvals from the community + one approval from a maintainer == PR can be merged

**Review by emmalearnscode — DISMISSED** [review]
Nice and clear, thanks for creating this. I asked Eddie if something like this existed a couple of weeks ago so great to see it getting written!

File: docs/moderators/github.md
Comment (kaiwalyakoparkar) [reply docs/moderators/github.md:9]: > I think we can also have a provision of -> Many approvals from the community + one approval from a maintainer == PR can be merged

This sounds good

**Review by BOLT04 — DISMISSED** [review]
LGTM 👍

**Review by Pradumnasaraf — DISMISSED** [review]

File: docs/moderators/github.md
Comment (Panquesito7) [inline docs/moderators/github.md:49]:
```suggestion
In general, only *one* maintainer should be reaching out to the contributor. If multiple maintainers reach out, we could inadvertently make our contributors feel pressured/rushed. We are all busy and want to respect each others' time.
```

File: docs/moderators/github.md
Comment (Panquesito7) [inline docs/moderators/github.md:55]:
```suggestion
If an issue should not be closed, remove the `stale issue` label that the action has added, and then leave a comment on the issue explaining why it is still necessary. If an issue is repeatedly being labeled by the bot, it may indicate that it is not a priority and we should re-evaluate the need.
```

Comment (nhcarrigan) [reply docs/moderators/github.md:55]: smh american spelling lol

**Review by Panquesito7 — APPROVED** [review]
Awesome! 🚀

**Review by schmelto — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]

**Review by krishguptadev — APPROVED** [review]

**Review by thepranaygupta — APPROVED** [review]

**Pull request merged**
